CLEARWATER-ENTERPRISE-MIB

```diff
@@ -0,0 +1,202 @@
+CLEARWATER-ENTERPRISE-MIB DEFINITIONS ::= BEGIN
+
+-- /**********************************************************/
```

Comment:
I think you can remove this comment - you changed all instances of ObjectID to get smilint passing.